core/templates/components/entity-creation-services/skill-creation.service.ts

```diff
@@ -77,8 +78,9 @@ angular.module('oppia').factory('SkillCreationService', [
         // as soon as the user clicks the 'Create' button and filled with URL
         // once the details are fetched from the backend.
         var newTab = $window.open();
+        var imagesData = ImageLocalStorageService.getStoredImagesData();
```

Comment:
Do get and flush in the same file/services (it can be here or in the directive which is calling this service)

Reply:
Done